packages/itinerary-body/src/types.ts

```diff
@@ -201,6 +201,8 @@ interface ItineraryBodySharedProps {
   setViewedTrip: SetViewedTripFunction;
   /** If true, will show agency information in transit legs */
   showAgencyInfo?: boolean;
+  /** If true, will show the effect date/time text in the alert body */
```

Comment:
Since it's default to true, maybe change the comment to "if false..."

Reply:
I agree. Good catch!